sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items-mlflow.ipynb

```diff
@@ -872,6 +872,79 @@
     "print(resp.text)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize detections\n",
```

Comment:
can we add the visualization section to all tasks same as v1?

Reply:
We need to add model deployment section as well before we can add visualization. 
There is another task to add model deployment and local mlflow inference sections to all tasks. Maybe this can be done as part of that task. What do you think ?

Reply:
I also think the same. First, we will need to add 2 notebooks for each task i.e., mlflow notebook having inference code and local inference notebook. Then, we can add visualization images to that class. It can be done as a part of that tasks.

Currently, object detection task is having these notebooks. So, I'm adding inferenced image visualization section in them. It can act as reference while adding to other tasks.